src/sql/base/browser/ui/panel/media/panel.css

```diff
@@ -12,6 +12,24 @@
 	flex-direction: column;
 }
 
+.vs-dark .tabbedPanel {
```

Comment:
the only attribute you need in the .vs-dark and .hc-black block is the border-top-color, the rest of attributes will be inherited from the .tabbedPanel